maintainers/maintainer-list.nix

```diff
@@ -5372,6 +5372,11 @@
     githubId = 852967;
     name = "Russell O'Connor";
   };
+  roelvandijk = {
+    email = "roel@lambdacube.nl";
+    github = "roelvandijk";
```

Comment:
Since https://github.com/NixOS/nixpkgs/pull/66361 you need to add `githubId` as well.